INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:4000
 * Running on http://172.18.0.4:4000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:routes.routes:Configure monitoring task request received
ERROR:root:'dict' object has no attribute 'read'
Traceback (most recent call last):
  File "/link-analyzer/controllers/monitoring_task_session_handler.py", line 73, in add_monitor_task
    monitor_obj = monitor_class.deserialize(config_json)
  File "/link-analyzer/models/analyses.py", line 176, in deserialize
    lstm_params_dict = json.load(json_string)
  File "/usr/local/lib/python3.9/json/__init__.py", line 293, in load
    return loads(fp.read(),
AttributeError: 'dict' object has no attribute 'read'
INFO:werkzeug:172.18.0.1 - - [13/Nov/2024 21:24:00] "[31m[1mPUT /monitor/config HTTP/1.1[0m" 400 -
INFO:routes.routes:Configure monitoring task request received
ERROR:root:'dict' object has no attribute 'read'
Traceback (most recent call last):
  File "/link-analyzer/controllers/monitoring_task_session_handler.py", line 73, in add_monitor_task
    monitor_obj = monitor_class.deserialize(config_json)
  File "/link-analyzer/models/analyses.py", line 176, in deserialize
    lstm_params_dict = json.load(json_string)
  File "/usr/local/lib/python3.9/json/__init__.py", line 293, in load
    return loads(fp.read(),
AttributeError: 'dict' object has no attribute 'read'
INFO:werkzeug:172.18.0.1 - - [13/Nov/2024 21:27:16] "[31m[1mPUT /monitor/config HTTP/1.1[0m" 400 -
